empire.go

```diff
@@ -169,9 +240,9 @@ func (e *Empire) DomainsDestroy(domain *Domain) error {
 	return e.domains.DomainsDestroy(domain)
 }
 
-// JobStatesByApp returns the JobStates for the given app.
-func (e *Empire) JobStatesByApp(ctx context.Context, app *App) ([]*ProcessState, error) {
```

Comment:
The terminology we used to describe running tasks was confusing/inconsistent before (jobs, job state, process state), so I'm just called these `Tasks'`s now.